src/plugins/index_pattern_field_editor/public/components/flyout_panels/flyout_panels_content.tsx

```diff
@@ -16,5 +16,5 @@ export const PanelContent: React.FC = (props) => {
     registerContent();
   }, [registerContent]);
 
-  return <div className="fieldEditor__flyoutPanel__content" {...props} />;
+  return <div className="fieldEditor__flyoutPanel__content" tabIndex={0} {...props} />;
```

Comment:
Adding a `tabIndex` prop to the generic `<Flyout.Content />` component as it is the body of the flyout which is scrollable. cc @myasonik